backend/src/components/sdc.js

```diff
@@ -304,6 +308,10 @@ async function updateAndValidateSdcSchoolCollectionStudent(req, res) {
     if (data?.enrolledProgramCodes) {
       data.enrolledProgramCodes = data?.enrolledProgramCodes.match(/.{1,2}/g);
     }
+
+    if (data?.numberOfCourses) {
```

Comment:
Looks like stripNumberFormatting should be used here @alexmcdermid

Reply:
I see now this is changing the return object, but I do think adding the stripNumberFormatting to node as well is a good idea